__tests__/lambdas/authorizer.js

```diff
@@ -0,0 +1,68 @@
+const mockClientId = process.env.CLIENT_ID
+
+import { handler } from '../../awslambda/Authorize'
+
+const mockMethodArn = 'Teleport'
+
+const mockValidToken = 'ricksanchez'
+const mockInvalidToken = 'mortysmith'
+
+jest.mock('axios', () => {
```

Comment:
Use `moxios` if you need to mock axios. Check the `app` actions test file for an example